public/splash.html

```diff
@@ -4,51 +4,80 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
+    <script>
+      if (typeof module === 'object') {
+        // unset module
+        window.module = module;
+        module = undefined;
```

Comment:
Can you explain this in more detail and why it's needed?

Reply:
Electron does not give its window module executable options that we are used to seeing in browser. If there wasn’t for this wrapper around function and variable definitions, bodymovin would be undefined. Same would happen if you were to add jquery or any other dependency